README.md

```diff
@@ -22,7 +22,46 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+### `params_pass`, `options_pass` and `attrs_pass`
+
```

Comment:
@emaglio How about renaming 
- `params_pass` to `default_params` ?
- `attrs_pass` to `expected_attributes` ?

That way, it would be easier to remember what params are being accessed when.

Reply:
100% with you! I was planning to change it.
@apotonick is still not 100% sure on this one though so it’s on hold this PR